packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -1,6 +1,7 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
```

Comment:
We want to use our own core-contracts library instead of OZ. You could argue that this case could be an exception, which would make sense, but my case is that that would make it hard for us to draw the line further down the road. So my suggestion is to only use OZ as reference.

Reply:
Agree.. let's bring the library that we need (and just what we use)